org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/NoActionsAvailableAction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2015 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
```

Comment:
Move the contributor section to the class-level javadoc.

Reply:
done